scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -214,9 +212,31 @@ public static BuildTargetResult CompileStage2(BuildTargetContext c)
                 }
             }
 
+            CleanOutputDir(Path.Combine(Dirs.Stage2, "sdk"));
```

Comment:
Actually I am a bit hesitant on cleaning out the PDBs from stage2 or stage1. How do we debug the dotnet CLI tools in our dev box with the PDBs missing?